python/samples/getting_started_with_agents/azure_ai_agent/step7_azure_ai_agent_retrieval.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from azure.identity.aio import DefaultAzureCredential
+
+from semantic_kernel.agents.azure_ai import AzureAIAgent
+
+"""
+The following sample demonstrates how to use an already existing
+Azure AI Agent within Semantic Kernel. This sample requires that you
+have an existing agent created either previously in code or via the
+Azure Portal (or CLI).
```

Comment:
maybe add a link to docs about how?

Reply:
Yes, it's already present in the README.

Reply:
it is not immediately obvious when seeing the description of this sample and here how I can do and especially where I can get my assistant id.